CHANGELOG.md

```diff
@@ -4,11 +4,19 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.4.1 (2021-08-19)
 
+This release contains bug fixes since the 2.4.0 release.
+
+We deem it high priority to upgrade since it is needed to support
+PostgreSQL 12.8 and 13.4.
 
 **Bugfixes**
 * #3430 Fix havingqual processing for continuous aggregates
+* #3468 Disable tests by default if tools are not found
+* #3468 Fix crash while tracking alter table commands
+* #3494 Improve error message when adding data nodes
+* #3498 Fix continuous agg bgw job failure for PG 12.8 and 13.4
```

Comment:
```suggestion
* #3489 Fix continuous agg bgw job failure for PG 12.8 and 13.4
* #3494 Improve error message when adding data nodes
```

Reply:
Thanks for catching it. That is indeed the case.